test-suite/src/dydx_airdrop/tests/test_claim.rs

```diff
@@ -37,6 +37,7 @@ fn query_minter_as_expected(app: &mut StargazeApp, airdrop_contract: Addr, minte
 fn test_instantiate() {
     let mut app = custom_mock_app();
     let minter_address = Addr::unchecked("contract1");
+    let name_discount_wl_address = Addr::unchecked(MOCK_NAME_DISCOUNT_WL_ADDR_STR);
```

Comment:
I think we need some new unit tests that test the additional functionality, such as can only do the claim after name is registered correct?

Reply:
Correct. Updating them currently.